.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## PR TITLE
+> Please fill out all values and then remove any help text before submitting your PR.
+> You can use [this PR](https://github.com/CrowdStrike/falconpy/pull/67) as a reference example.
+
+- [ ] Enhancement
+- [ ] Major Feature update
+- [ ] Bug fixes 
+- [ ] Breaking Change
+- [ ] Updated unit tests
+- [ ] Additional documentation required
+
+> Check the values above that match your PR and remove the remaining.
+
+```shell
+COVERAGE RESULTS SHOULD BE POSTED HERE
+```
+
+## Added features and functionality
++ Details of new features and functionality should be listed here
+
+## Issues resolved
++ Bug fix: Bugs should be listed individually. Please link your PR to any related issues so that they are triaged properly. Add a link to this line to the issue.
```

Comment:
```suggestion
+ Bug fix: Please list related bugs individually. Identifying the ticket or bug report in the PR description auto-updates the affected ticket and helps the community with ticket triage. For example:

* Fixes https://github.com/CrowdStrike/falconpy/issues/1234 by doing foo 
* Mitigates https://github.com/CrowdStrike/falconpy/issues/5678 by updating bar.
```

Reply:
* Saying please first is just polite!
* Gave examples of how to call out tickets